lib/plugins/fruitdown/config.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+module.exports = {
+  name: 'fruitdown',
+  valid: function () {
+    return !!global.indexedDB;
```

Comment:
I would say `return !!global.indexedDB && !!global.IDBKeyRange` (just to be safe in case they are testing in a weirdo old Samsung/HTC Android phone)